### PR TITLE
Get active count

### PR DESCRIPTION
**Pull request by saisankargochhayat**
## Related Issues and Dependencies
Depended on #1768

Method to return the total count of active repositories.

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -3606,6 +3607,65 @@ def _runtime_environment_conf2models(
...
+        with self._session_scope() as session:
+            instance = session.query(KebechetGithubAppInstallations)\
+                .filter(KebechetGithubAppInstallations.slug == slug).first()
+            if instance:
+                instance.is_active = False
+                session.commit()
+                return True
+            return False
+
+    def get_active_kebechet_installations(self) -> int:
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py:3662]: please follow https://github.com/thoth-station/storages/blob/master/docs/conventions/README.md convetions for query name :)

Comment (saisankargochhayat) [reply thoth/storages/graph/postgres.py]: `get_kebechet_github_installations_count_per_is_active` this is what I could derive, is that right? :smiley:

**Comment by goern** [comment]
LGTM

**Comment by saisankargochhayat** [comment]
recheck

**Comment by saisankargochhayat** [comment]
regate

**Comment by saisankargochhayat** [comment]
regate

**Comment by saisankargochhayat** [comment]
recheck

**Comment by goern** [comment]
recheck

**Comment by goern** [comment]
regate

**Pull request merged**